pkg/cmd/login/login.go

```diff
@@ -46,6 +46,13 @@ func (*LoginController) AuthenticateToOktetoCluster(ctx context.Context, oktetoU
 	if token == "" {
 		oktetoLog.Infof("authenticating with browser code")
 		user, err := WithBrowser(ctx, oktetoURL)
+		// If there is a TLS error, return the raw error
+		if err != nil && oktetoErrors.IsX509(err) {
+			return nil, oktetoErrors.UserError{
+				E:    err,
+				Hint: "Add the flag \"--insecure-skip-tls-verify\" to connect to an instance with self-signed certificates",
```

Comment:
```suggestion
				Hint: "Add the flag '--insecure-skip-tls-verify' to connect to an instance with self-signed certificates",
```
I think we are using single quotes in other info messages